community/kernel/src/main/java/org/neo4j/graphdb/factory/GraphDatabaseSettings.java

```diff
@@ -375,47 +374,7 @@
                   "the page cache. The default page cache memory assumes the machine is dedicated to running " +
                   "Neo4j, and is heuristically set to 50% of RAM minus the max Java heap size." )
```

Comment:
Now with default value as null with evaluation in a factory it's not clear from here what is a default value and where it's get calculated. 
As soon as you moved default value calculation from here to ConfiguringPageCacheFactory that makes that settings special in terms how we calculate default value and make it less clear.